backends/mlu/kernels/matmul_kernel.cc

```diff
@@ -460,7 +460,43 @@ void MatmulWithFlattenGradKernel(const Context& dev_ctx,
   }
 }
 
+template <typename T, typename Context>
+void BmmKernel(const Context& dev_ctx,
+               const phi::DenseTensor& x,
+               const phi::DenseTensor& y,
+               phi::DenseTensor* out) {
+  MatMulND<T>(dev_ctx, x, y, out, false, false);
```

Comment:
Is it necessary to check the shape first?

Reply:
Personally, I think it is unnecessary